.github/workflows/ci.yml

```diff
@@ -2,7 +2,16 @@ name: CI
 
 on:
   push:
+    branches:
+      - main
+      - ci-proposal
```

Comment:
I'm guessing the idea would be to remove this line before merge, once we're happy that it works?